src/javascripts/services/transactionService.js

```diff
@@ -48,6 +47,12 @@ class TransactionService {
     updateTransaction = async(data) => {
         const transaction = new Transaction(data);
         await this.api.updateTransaction(transaction);
+        this.transactionList = this.transactionList.map((item) => {
+            if (item.id == transaction.id) {
```

Comment:
Should not use ==, use === instead. For different types, pls use parse

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!